app/posts/2021-08-31-scope-expansion-round-2.md

```diff
@@ -0,0 +1,240 @@
+
+It varied amongst hiring staff whether working a full school day was classed as full time or part time, resulting in job listings with the same required hours being listed under different working patterns.
+
+> "I think this might be slightly confusing because if someone applies for a full time job at a school, they are going to assume they will be working all day”
+
+Hiring staff rarely selected more than one working pattern, despite describing the roles they were advertising as both term time and full/part time.
+
+Hiring staff advertise term time roles as ‘term time +_’  to cover things like paid training days.
+
+### Pay Package
+
+If the term time options include either Part time, Job share or Term time, we show a new field, Actual salary, on the Pay package step.
+
+We included the actual salary as we found it can be both time consuming and frustrating for jobseekers to have to work out their actual earnings from the full time equivalent (FTE) or grade scale provided. This can be misleading, particularly for lower paid support roles.  
+
+{{ appFigure({
+  image: {
+    file: "step4.png",
+    alt: "The pay package screen in the create a job flow"
+  }
+}) }}
+
+### How did this test?
+
+Half of the participants would use the actual salary field as they felt it was important to give jobseekers clarity on their earnings. The remaining half would have left this blank. They either felt seeing a lower salary could put applicants off or felt they (or their current systems) would not accurately work this out.
+
+> "I never quite trust our budgeting software to get it right so I wouldn’t put that in"
+
+### Applying for the job
+
+The initial design included information explaining that hiring staff aren't able to accept education support applications through the service at this time. We removed this information after receiving some feedback.
+
+{{ appFigure({
+  image: {
+    file: "step6 original.png",
+    alt: "The applying for the job screen in the create a job flow"
+  }
+}) }}
+
+### How did this test?
+
+Telling hiring staff they cannot accept applications directly through Teaching Vacancies was problematic. Some believed this meant education support roles cannot be advertised and therefore thought they could go no further. As none of the participants were aware of the Teaching Vacancies application form and tracking system prior to this they were not clear on what it was or when it would be available.  
+
+> “When I draft an education support vacancy listing, it prompts me to choose how applicants should apply and advises that it is not possible for candidates to apply directly through teaching services. I do not wish for direct applications, but it will not permit me to go further with the form.”
+
+* It wasn’t clear to hiring staff how they could include a link to an online application form.
```

Comment:
This is weird being a bullet point now that none of the other lines are bullet points!